incubator/sentry/templates/web-deployment.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+spec:
+  replicas: {{ .Values.webReplicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+        role: web
+    spec:
+      containers:
+      - name: {{ .Chart.Name }}
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
```

Comment:
Thanks, I initially tried using the log in page as a health check but removed it because I didn't look up what endpoint should be checked instead. I'll add this along with better limits.